web/frontend/src/pages/form/components/utils/countResult.ts

```diff
@@ -42,31 +42,20 @@ const countRankResult = (rankResult: number[][], rank: RankQuestion) => {
 // percentage of the total number of votes and which candidate(s) in the
 // select.Choices has the most votes
 const countSelectResult = (selectResult: number[][]) => {
-  const resultsInPercent: string[] = [];
-  const maxIndices: number[] = [];
-  let max = 0;
-
-  const results = selectResult.reduce((a, b) => {
-    return a.map((value, index) => {
-      const current = value + b[index];
-
-      if (current >= max) {
-        max = current;
-      }
-      return current;
+  let results: [string, number][] = [];
+
+  selectResult
+    .reduce(
+      (tally, currBallot) => tally.map((currCount, index) => currCount + currBallot[index]),
+      new Array(selectResult[0].length).fill(0)
+    )
+    .forEach((totalCount) => {
+      results.push([
+        (Math.round((totalCount / selectResult.length) * 100 * 100) / 100).toFixed(2).toString(),
+        totalCount,
+      ]);
```

Comment:
Or remove the `let result` altogether and do something like:

```suggestion
  return selectResult
    .reduce(
      (tally, currBallot) => tally.map((currCount, index) => currCount + currBallot[index]),
      new Array(selectResult[0].length).fill(0)
    )
    .map((totalCount) => 
      [
        (Math.round((totalCount / selectResult.length) * 100 * 100) / 100).toFixed(2).toString(),
        totalCount,
      ]);
```